Runtime/Runner/Scenes/Localization/GetSimvaStringName.cs

```diff
@@ -0,0 +1,38 @@
+﻿using UnityEngine;
+using UnityEngine.UI;
+
+namespace Simva
+{
+    public class GetSimvaStringName : MonoBehaviour
+{
+
+    private void Awake()
+    {
+        FillName();
+    }
+
+    void FillName()
```

Comment:
Instead of searching for the text object or text mesh, it would be much simpler and clear to have a reference to the text/textmesh component that it's supposed to change. 
Also, I discourage you to use the GameObject.name as the "key" to get the language element and instead add a
```
 [SerializeField] private string languageKey;
```
or similar.

It will be less error prone.

Reply:
fixed